### PR TITLE
PS-9333 5.7.44-51 - 5.7 Post-EOL 3 release notes and release note tasks

### PR DESCRIPTION
**Pull request by patrickbirch**
On branch ps-9333
	new file:   docs/release-notes/5.7.44-51.md
	modified:   docs/release-notes/release-notes_index.md
	modified:   mkdocs-base.yml
	modified:   variables.yml

### CODE REVIEW

File: docs/release-notes/5.7.44-51.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+### Non-security fixes
+
+* [Bug #35205358]: The `conn_delay/Waiting` stage in the connection_control plugin was not reset after a delay caused by the plugin, leading to inaccurate monitoring data.
+
+* [Bug #34929814]: Running two OPTIMIZE TABLE commands at the same time on a table with fulltext indexes and innodb_optimize_fulltext_only enabled sometimes made the server shut down.
+
+* [Bug #35894664]: The result from a spatial index that included a column with a spatial reference identifier (SRID) was empty. Additionally, using FORCE INDEX to enforce a covering index scan on the spatial index caused an assertion error.
+
+* [Bug #331426544, #36543458]: Adds an additional check on the QT_NO_TABLE flag before accessing table_name.
```
Comment (alina-derkach-oaza) [inline docs/release-notes/5.7.44-51.md]: There are too bugs. So maybe change to 

[`Bugs` #331426544, #36543458]: `Add` an additional check on the QT_NO_TABLE flag before accessing table_name.

**Review by alina-derkach-oaza — APPROVED** [review]

**Comment by VarunNagaraju** [comment]
Hi @patrickbirch, If you take a look at https://github.com/percona/percona-server/commits/release-5.7.44-51/, there is a commit https://github.com/percona/percona-server/commit/29c2ef53acae5c374b23322e860b5c1f6f5a92b6 which is a prerequisite for https://github.com/percona/percona-server/commit/0df51ea69ee1424466cc5203927f88c1a09b37c8. So, please document that as well.
`For the CONNECTION_CONTROL plugin, the Performance Schema instrumentation used keys that were not discoverable to the Performance Schema unless the associated code actually executed. (Bug #29539976)`
https://dev.mysql.com/doc/relnotes/mysql/8.0/en/news-8-0-20.html

**Review by VarunNagaraju — APPROVED** [review]

File: docs/release-notes/5.7.44-51.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+[Contribute to the documentation](https://github.com/percona/psmysql-docs/blob/8.0/contributing.md)
+
+For [training](https://www.percona.com/training), contact [Percona Training - Start learning now](https://learn.percona.com/contact-me)
+
+
+[MySQL 5.7 Post-EOL Support from Percona customers]: https://www.percona.com/post-mysql-5-7-eol-support
+
+[build this release from the source]: ../installation/git-source-tree.md
+
+[Bug #36563773]: https://github.com/mysql/mysql-server/commit/36563773
```
Comment (sleto-it) [inline docs/release-notes/5.7.44-51.md]:
```suggestion
[Bug #36563773]: https://github.com/mysql/mysql-server/commit/2f67801d010
```

File: docs/release-notes/5.7.44-51.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+[Bug #35205358]: https://github.com/mysql/mysql-server/commit/35205358
```
Comment (sleto-it) [inline docs/release-notes/5.7.44-51.md]:
```suggestion
[Bug #35205358]: https://github.com/mysql/mysql-server/commit/3fc9c6a63a2
```

**Review by sleto-it — COMMENTED** [review]
I think there's still something that does not convince me. 

1. The Bug number #331426544.  It is actually #33142654 without the last 4.

2. Depending on what we want to do here: 
- Option 2.1. We can consider to mention the two bugs in separate lines. One bug is "Bug#36543458 Bug#33142654 needs additional patch for MacOS " and the other is " Bug #33142654 THE SECOND HEAP-USE-AFTER-FREE ISSUE OF MYSQL SERVER". They have two different links https://github.com/mysql/mysql-server/commit/76dd59535fb and https://github.com/mysql/mysql-server/commit/227ffcd4dad 
- Option 2.2. We can consider to combine the two bugs into the same line, as they are related (with two different links)?

3. 33057164 has also two different links (the original fix and the post-fix).  Again unsure how we want to handle, but we can consider putting both the links into a single line. It depends on our goals. If we want to mention all the commits, good to have the separate links (in the internal spreadsheet we keep different lines, with different links).

@VarunNagaraju in case you have option.

Hope this helps.  Other than this - I think it is OK.  Thanks

**Comment by VarunNagaraju** [comment]
Hi @patrickbirch, Both the bugs in a single line are fine since the description for both bugs are the same. I'd request the following changes
1. The trailing 4 from the first bug number should be removed.
2. The two bug numbers should point to different links. Bug #33142654 https://github.com/mysql/mysql-server/commit/227ffcd4dad and Bug#36543458 https://github.com/mysql/mysql-server/commit/76dd59535fb
3. Add the phrase "to fix HEAP-USE-AFTER-FREE errors" at the end of the single line description.

What do you think @sleto-it?

**Review by VarunNagaraju — APPROVED** [review]
LGTM

**Review by sleto-it — APPROVED** [review]
Thanks

**Pull request merged**